### PR TITLE
[MPD][Documentation] Fix an error and change a bit of wording

### PR DESCRIPTION
**Pull request by jpmens**
`s/Exec/MPD` and change a bit of the wording. Also add download link to `.jar` file.

### CODE REVIEW

File: bundles/binding/org.openhab.binding.mpd/README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 [Music Player Daemon (MPD)](http://www.musicpd.org/) is a flexible, powerful, server-side application for playing music. Through plugins and libraries it can play a variety of sound files while being controlled by its network protocol. 
 
-By help of the openHAB MPD binding, you can e.g. start/stop playing music in specific rooms / on various channels and change volume.
+With the openHAB MPD binding you can e.g. start/stop playing music in specific rooms / on various channels, change volume, and even see which track is playing.
 
 ## Binding Configuration
 
-This binding can be configured in the file `services/mpd.cfg`.
+After installing the MPD version 1 binding by [downloading the .jar file](https://openhab.ci.cloudbees.com/job/PR-openHAB1-Addons/ws/bundles/binding/org.openhab.binding.mpd/target/) into the `addons/` directory, the binding can be configured in the file `services/mpd.cfg`.
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.mpd/README.md]: This is about configuration, not installation. Please revert.

Comment (jpmens) [reply bundles/binding/org.openhab.binding.mpd/README.md]: so where is installation of MPD explained?

**Comment by jpmens** [comment]
That doesn't really answer my question. I asked _where is the installation of mpd explained_. I realize there is a generic installation instruction, and I also know there are many postings containing incorrect and outdated information in the forums.

For what it's worth, I started providing patches to documentation hoping that fixing a word or two wouldn't entail long arguments as to why and why not and wherefore, and it's difficult for me to understand why we shouldn't make it easier for people to get started, but if that's policy, I will stand down

I am quickly loosing interest in helping improve the openHAB documentation.

For your perusal here are a few snippets of other pages in this documentation set. 

## Snippets from random binding pages


[amazon dash](https://www.openhab.org/addons/bindings/amazondashbutton/#prerequisites)

> Start with installing libpcap (for Mac/Linux/UNIX) or WinPcap (for Windows) on your computer  yada yada sudo apt-get install yada yada

[piface](https://www.openhab.org/addons/bindings/piface1/)

> please install the scripts that can be found here. If you upgrade from an older openHAB version make sure that you update the scripts to avoid watchdog errors in your

> e RasPi firmware (rpi-update command) even if the the ohpiface scripts are correctly installed and whe

> lsmod | grep spi_bcm2708

[serial](https://www.openhab.org/addons/bindings/serial1/)

> adapt start.sh to have the serial port included. The java command line should then include the following parameters: -Dgnu.io.rxtx.SerialPorts=/dev/ttyAMA0

[zigbee](https://www.openhab.org/addons/bindings/zigbee/)

> flashed with the correct firmware in order to work with this binding. The file can be downloaded from TI website archives (http://www.ti.com/tool/z-stack-archive) as part of the Z-STACK-HOME v.1.2.2a package. The file name is CC2531ZNP-Pro-

[caldav](https://www.openhab.org/addons/bindings/caldav-command1/)

> First of all you need to add the org.openhab.io.caldav-version.jar to the addons folder.

[caldav personal](https://www.openhab.org/addons/bindings/caldav-personal1/)

>  First of all you need to add the org.openhab.io.caldav-version.jar to the addons folder.

[allplay](https://www.openhab.org/addons/bindings/allplay/)

> libraries have a dependency on the Visual C++ Redistributable for Visual Studio 2015. If you are using Windows, please make sure to install these components

**Comment by jpmens** [comment]
@9037568 I may have hit close by mistake

**Comment by 9037568** [comment]
>That doesn't really answer my question.

It does, actually. The installation process for MPD is the same as for every other generic binding, and is therefore fully explained in the link I provided.  You'll notice most of the binding quotes you provided above are giving detailed instructions that apply ONLY to that one binding.

So my position remains that adding redundant text here about installation should not be done.

**Comment by 9037568** [comment]
Thanks for the enhancements, @jpmens !

**Pull request merged**